test/content.test.js

```diff
@@ -30,10 +30,6 @@ describe(modulePath, () => {
 
 
   describe('specificValues option', () => {
-    it('checks specified value to be present', () => {
```

Comment:
I think this test can't be removed, right? Otherwise, you never prove that a content from the session can end up in the view via the content json. Is that right?